ceno_zkvm/examples/fibonacci_elf.rs

```diff
@@ -269,18 +295,16 @@ fn main() {
     }
 
     let transcript = Transcript::new(b"riscv");
-    // change public input maliciously should cause verifier to reject proof
+    // Maliciously changing the public input should cause the verifier to reject the proof.
     zkvm_proof.raw_pi[0] = vec![<GoldilocksExt2 as ff_ext::ExtensionField>::BaseField::ONE];
     zkvm_proof.raw_pi[1] = vec![<GoldilocksExt2 as ff_ext::ExtensionField>::BaseField::ONE];
 
-    // capture panic message, if have
-    let default_hook = panic::take_hook();
```

Comment:
@lispc Here's an example where I had some (minor) trouble understanding what the original code tried to do.  So I cleaned it up into its own helper function, in order to make the job of the next reader down the line a bit easier.